test/runtime/configMatters/forall-unordered-opt/COMPOPTS

```diff
@@ -0,0 +1 @@
+--report-optimized-forall-unordered-ops
```

Comment:
We'll have to be careful about how many tests we're adding to runtime/configMatters. We are somewhat limited on how much testing we can do on crays.

Reply:
I might be able to merge some tests but I don't see how else to do this because the optimization won't optimize local atomics (so the reporting will be wrong).

I suppose I could make the optimization print out if it considered optimizing a local atomic (ignoring the fact that the runtime does nothing solely for testing) and in that event we could move the compile-time-reporting tests out of this directory. Do you think that'd be an improvement?